main.js

```diff
@@ -0,0 +1,44 @@
+let bookshell = [];
+
+const addbtn = document.querySelector('#addBtn');
+const BookListContainer = document.querySelector('.books-container');
+const localData = JSON.parse(localStorage.getItem('Booked'));
+
+if (localData != null) {
+  localData.forEach((item) => {
+    BookListContainer.innerHTML += `
+    <div>
+    <div>${item.name}</div>
+    <div>${item.author}</div>
+    <button class='btn-remove' id='${item.id}' onclick='removeBook(${item.id})'>Remove</button>
+    </div>
+    <hr>
+    `;
+    const book = {
+      id: item.id,
+      name: item.name,
+      author: item.author,
+    };
+    bookshell.push(book);
+  });
+}
+
```

Comment:
- With your current implementation you are reloading the webpage everytime you add and delete a book, this is not an efficient way of updating the overall data of your webpage rather you can adopt the method of creating a function which would display the new list of books after adding and removing a book. 👍